docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -644,6 +644,15 @@ The entry of the hashtable is a counted sequence of compressed unsigned integers
 
 This section may be included in addition to a InliningInfo2 section.
 
+## ReadyToRunSectionType.HotColdMap (v8.0+)
+In ReadyToRun 8.0+, the format supports splitting a method into hot and cold parts so that they are not located together. This hot-cold map section captures the information about how methods are split so that the runtime can locate them for various services.
```

Comment:
Add the detail in line 180 about what the ReadyToRun section id is for the hot cold map.